.gitlab-ci.yml

```
@@ -16,6 +18,15 @@ variables:
   # change to production after https://github.com/paritytech/ci_cd/issues/535 is closed
   CI_IMAGE:                        "paritytech/ci-linux:staging"
 
+.test-refs:                        &test-refs
```

```suggestion
.common-refs:                      &common-refs
```
I would rename it since jobs run in all branches

---

Changed